dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -115,7 +115,7 @@ def valid_scripts
     return head :forbidden unless current_user
 
     scripts = Script.valid_scripts(current_user).map(&:assignable_info)
-    render json: {scripts: scripts}
+    render json: scripts
```

Comment:
please double check that all of your integers in this json are coming back as integers and not strings. for some reason, I ran into a problem when rendering an array as json (like you have it here), rather than rendering an object (like it was before). past discussion for a bit of context: https://github.com/code-dot-org/code-dot-org/pull/22176#discussion_r185861050

Reply:
thank you for the heads up! i double checked my response and the integers look good (image from the frontend below), but i'll keep an eye out for any weirdness with that

<img width="727" alt="screen shot 2018-05-24 at 2 52 33 pm" src="https://user-images.githubusercontent.com/9812299/40515628-6c3f75b0-5f62-11e8-8e87-5764c5001d4b.png">